addons/mass_mailing/models/mailing_list.py

```diff
@@ -13,7 +13,7 @@ class MassMailingList(models.Model):
 
     name = fields.Char(string='Mailing List', required=True)
     active = fields.Boolean(default=True)
-    contact_nbr = fields.Integer(compute="_compute_contact_nbr", string='Number of Contacts')
+    contact_nbr = fields.Integer(compute="_compute_contact_nbr", string='Number of Contacts', default=0)
```

Comment:
Actually "default" should be part of the compute itself, so that it is well contained. Purpose is that method always gives a valid result and does not depend from other field attributes.

In compute, just do: if self.ids -> computation, otherwise -> self.contact_nbr = 0 .

Reply:
That's funny because it was my first fix, but I thought it was more "clean" like this.
Does that mean that we should never use the default attribute on a compute field ?

Reply:
Indeed. Either it is computed, either you can set it manually (with maybe a default value that is "ok for most users").